packages/mwp-app-render/src/components/uxcapture/UXCaptureImageLoad.jsx

```diff
@@ -0,0 +1,66 @@
+// @flow
+import * as React from 'react';
+
+type Props = React$ElementConfig<typeof HTMLImageElement> & { mark: string };
+
+export const getOnLoadJS = (mark: string): string => {
+	const onload = `
+		if (window.UX && !window.UX['${mark}-LOADED']) {
+			window.UX.mark('${mark}');
+			window.UX['${mark}-LOADED'] = true;
+		}
+	`;
+
+	// Replace newlines and tabs with space characters
+	return onload.replace(/[\n\t]+/g, ' ');
+};
+
+export const getPropsAsHTMLAttrs = (
+	props: React$ElementConfig<typeof HTMLImageElement>
+): string => {
+	return Object.keys(props)
+		.map(prop => {
+			let attribute = prop;
+
+			// convert JSX attribute names to HTML syntax
+			// Any other special props needed? Add here.
+			// @see: https://reactjs.org/docs/dom-elements.html
+			if (prop === 'className') {
+				attribute = 'class';
+			}
+			if (prop === 'htmlFor') {
+				attribute = 'for';
+			} else {
+				// handles camelCased attributes like `onLoad`
+				attribute = attribute.toLowerCase();
```

Comment:
this won't work with aria attributes https://reactjs.org/docs/accessibility.html#wai-aria

Reply:
Not sure I follow for the aria attributes. Aria attributes are `aria-*` in JSX and remain the same in html.